### PR TITLE
chore(): add life-events-ds package

### PR DESCRIPTION
**Pull request by matteolc**
### Description

Add shared @ogcio/life-events-ds package

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**
- [ ] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### CODE REVIEW

File: packages/design-system/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-image": "^3.0.3",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-replace": "^5.0.5",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-typescript": "^11.1.6",
+    "@types/react": "18.3.12",
+    "@types/react-dom": "18.3.1",
+    "install": "^0.13.0",
+    "npm": "^10.4.0",
```
Comment (alewin) [inline packages/design-system/package.json:36]:
```suggestion
```
i suppose it's an `npm install` typo?

**Review by alewin — APPROVED** [review]

File: packages/life-events-ds/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@ogcio/life-events-ds",
+  "version": "0.1.0",
+  "description": "OGCIO Design System for Life Events",
+  "type": "module",
+  "main": "dist/esm/index.js",
```
Comment (SamSalvatico) [inline packages/life-events-ds/package.json:6]: you can remove main if `exports` is set

File: packages/life-events-ds/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "types": "./dist/index.d.ts"
+    },
+    "./styles": "./dist/style.css",
+    "./esm/styles": "./dist/esm/index.css",
+    "./cjs/styles": "./dist/cjs/index.css"
+  },  
+  "files": ["dist"],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "build": "rm -rf dist && rollup -c"
```
Comment (SamSalvatico) [inline packages/life-events-ds/package.json:22]: please add prepublish

File: packages/life-events-ds/package.json (hunk 2 above)
Comment (matteolc) [reply packages/life-events-ds/package.json:6]: https://github.com/ogcio/shared-node-utils/commit/54d6b934921af48e874187d8784756e5dbf0d9d6

File: packages/life-events-ds/package.json (hunk 3 above)
Comment (matteolc) [reply packages/life-events-ds/package.json]: https://github.com/ogcio/shared-node-utils/commit/080afdf17e40aa7737de9c49d0c45df3df9e3eef

**Review by SamSalvatico — APPROVED** [review]

File: packages/life-events-ds/package.json (hunk 2 above)
Comment (matteolc) [reply packages/life-events-ds/package.json:6]: Actually rollup needs a `main` :)

**Pull request merged**